components/brave_wallet/browser/solana_message.cc

```diff
@@ -168,6 +169,74 @@ absl::optional<std::vector<uint8_t>> SolanaMessage::Serialize(
   return message_bytes;
 }
 
+// static
+absl::optional<SolanaMessage> SolanaMessage::Deserialize(
```

Comment:
for reference https://github.com/solana-labs/solana-web3.js/blob/7f33484b2e1160a1bbecb9dc330252d6f7b1a4aa/src/message.ts#L221